services/web/server/src/simcore_service_webserver/exporter/formatters/sds/xlsx/templates/submission.py

```diff
@@ -0,0 +1,75 @@
+from typing import List, Tuple, Dict
+
+from pydantic import BaseModel, Field, StrictStr
+from simcore_service_webserver.exporter.formatters.sds.xlsx.xlsx_base import (
+    BaseXLSXCellData,
+    BaseXLSXSheet,
+    BaseXLSXDocument,
+)
+from simcore_service_webserver.exporter.formatters.sds.xlsx.styling_components import (
+    T,
+    TB,
+    Backgrounds,
+    Borders,
+)
+from simcore_service_webserver.exporter.formatters.sds.xlsx.templates.utils import (
+    ensure_correct_instance,
+)
+
+
+class SubmissionDocumentParams(BaseModel):
+    award_number: StrictStr = Field(
+        "", description="Grant number supporting the milestone"
+    )
+    milestone_archived: StrictStr = Field(
+        "", description="From milestones supplied to NIH"
+    )
+    milestone_completion_date: StrictStr = Field(
+        "",
+        description=(
+            "Date of milestone completion. This date starts the countdown for submission "
+            "(30 days after completion), length of embargo and publication date (12 "
+            "months from completion of milestone)"
+        ),
+    )
+
+
+class SheetFirstSubmission(BaseXLSXSheet):
```

Comment:
So if I understand well, the data is in pydantic models and then it can be mapped into excel sheets, right?
can we perhaps separate both concepts in different files such that e.g. one day we might decide to export in json instead of excel. Then we separate the data from the export data ...

Reply:
The reason why these pydantic validation models are in the same file is because they are used to validate only these excels. They are not intended to be used elsewhere and I would not sparate them.

I'd split them only if we actually need such a feature.